deployer/src/config_creator.py

```diff
@@ -137,7 +166,8 @@ def create_config(u=None):
         cuid = helpdesk_helper.get_conversation_ID_from_url(u)
 
         conversation = helpdesk_helper.get_conversation(cuid)
-        url_from_conversation = helpdesk_helper.get_start_url_from_conversation(conversation)
+        url_from_conversation = helpdesk_helper.get_start_url_from_conversation(
```

Comment:
Same as above, the ending character should be on a new line:
```python
url_from_conversation = helpdesk_helper.get_start_url_from_conversation(
    conversation
)
```

Reply:
Code style discussions can be a never-ending process. Whatever we choose, there are two important things:
1.The styling should make diffing the file easy. If we only fixed a typo in a comment, or we added an element to an array, or added a new argument to a function, the diff (on GitHub or on the terminal) should show us those changes easily. We don't want the whole line to be shown as changed if we only changed a tiny bit of it.
2. We need a tool that does the formatting for us. Aligning stuff, adding new lines is tedious and does not require human brainpower. We need something that can do it for us by the press of a button or in a git hook automatically. This will give us consistency and will help us all own the code as a whole as there will no longer be a "this part of the code was written by X", and "this one was written by Y" because you won't be able to see who wrote what, they will follow the same style.

So. For this PR, let's not spend too much time on the correct way of styling things. We'll need another PR that will add the required tooling to do this automatically for us on each hook. I personally recommend using `black` for auto-formatting, and `flake8` for linting issues that can't be  solved by `black`. But this should come into another PR.